tools/clang/lib/SPIRV/LowerTypeVisitor.cpp

```diff
@@ -972,6 +981,66 @@ LowerTypeVisitor::generateFunctionInfo(const CXXMethodDecl *decl,
   return fn;
 }
 
+SpirvDebugTypeComposite *LowerTypeVisitor::lowerDebugTypeComposite(
```

Comment:
It looks like this function has a lot of duplicate code from the other `lowerDebugTypeComposite` function.

Reply:
Right. Let me consider how to refactor it.